src/index.html

```diff
@@ -6,7 +6,37 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
-</html>
+  <body class="month">
```

Comment:
```suggestion
  <body>
     <div class="month ..."> 
```

it's better to create container for calendar

Reply:
also `start-day` and `month-length` modifiers must be implemented in the calendar according to the task